app/src/androidTest/java/org/liberty/android/fantastischmemo/test/ui/QuizHistoryTest.java

```diff
@@ -0,0 +1,96 @@
+//package org.liberty.android.fantastischmemo.test.ui;
```

Comment:
what is this espresso test, testing exactly?

Reply:
that it goes to the history page